cmd/remotefs/main.go

```diff
@@ -72,7 +76,7 @@ func main() {
 
 	level, err := logrus.ParseLevel(*logLevel)
 	if err != nil {
-		logrus.Fatal(err)
+		logrus.Fatalf("Failed to parse logLevel: %s\n"+ERROR_STRING, err)

```

Comment:
would it be better to do:
`logrus.Fatalf("Failed to parse logLevel: %s\n%s", err, ERROR_STRING)` ?

Reply:
Yeah I thought about that after the fact but decided to wait until today to see what you guys thought too